index.html

```diff
@@ -111,13 +111,27 @@
     color: var(--secondary-text-color);
 }
 
-.links {
+#demo-list .links {
     border-top: 1px var(--border-color) solid;
     padding: 8px 8px 0 8px;
     vertical-align: middle;
     margin: 16px 0 0 -8px;
 }
 
+.tool {
+    margin: 2em 2em 4em 2em;
```

Comment:
I don't think we should special-case the tools; they should have the same style/margin as the rest